{"uuid": "f70a0ec2-92af-4417-8c5d-5773924dcf78", "children": ["0048240e-e517-4ced-8baa-cf845f8ac336"], "befores": [{"name": "mobile_management", "status": "broken", "statusDetails": {"message": "TypeError: expected str, bytes or os.PathLike object, not NoneType\n", "trace": "  File \"/Users/bella/projects/qa_guru_10.22/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/bella/projects/qa_guru_10.22/.venv/lib/python3.9/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/Users/bella/projects/qa_guru_10.22/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/bella/projects/qa_guru_10.22/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/bella/projects/qa_guru_10.22/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/bella/projects/qa_guru_10.22/tests_app/conftest.py\", line 25, in mobile_management\n    options = config_app.to_driver_options(context=context)\n  File \"/Users/bella/projects/qa_guru_10.22/config.py\", line 45, in to_driver_options\n    options.set_capability('app', path.relative_from_root(self.APP))\n  File \"/Users/bella/projects/qa_guru_10.22/utils/path.py\", line 5, in relative_from_root\n    return Path(__file__).parent.parent.joinpath(path).absolute().__str__()\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pathlib.py\", line 972, in joinpath\n    return self._make_child(args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pathlib.py\", line 737, in _make_child\n    drv, root, parts = self._parse_args(args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pathlib.py\", line 691, in _parse_args\n    a = os.fspath(a)\n"}, "steps": [{"name": "Set options", "status": "broken", "statusDetails": {"message": "TypeError: expected str, bytes or os.PathLike object, not NoneType\n", "trace": "  File \"/Users/bella/projects/qa_guru_10.22/tests_app/conftest.py\", line 25, in mobile_management\n    options = config_app.to_driver_options(context=context)\n  File \"/Users/bella/projects/qa_guru_10.22/config.py\", line 45, in to_driver_options\n    options.set_capability('app', path.relative_from_root(self.APP))\n  File \"/Users/bella/projects/qa_guru_10.22/utils/path.py\", line 5, in relative_from_root\n    return Path(__file__).parent.parent.joinpath(path).absolute().__str__()\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pathlib.py\", line 972, in joinpath\n    return self._make_child(args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pathlib.py\", line 737, in _make_child\n    drv, root, parts = self._parse_args(args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pathlib.py\", line 691, in _parse_args\n    a = os.fspath(a)\n"}, "start": 1713896943621, "stop": 1713896943621}], "start": 1713896943435, "stop": 1713896943623}], "start": 1713896943435, "stop": 1713896943826}